test/test_data/multiple_pass.py

```diff
@@ -0,0 +1 @@
+pass; pass
```

Comment:
Maybe add another line with a single `pass`, to exercise the flattening function?